draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -965,6 +957,7 @@ The following list contains the most important changes from the previous specifi
 * RFC6614 marked TLSv1.1 or later as mandatory, this specification requires TLSv1.2 as minimum and recommends usage of TLSv1.3.
 * RFC6614 allowed use of TLS compression, this document forbids it.
 * RFC6614 only requires support for the trust model "certificates with PKIX" ({{?RFC6614, Section 2.3}}).  This document changes this.  For servers, TLS-X.509-PKIX ({{tlsx509pkix}}, equivalent to "certificates with PKIX" in RFC6614) and TLS-PSK ({{tlspsk}}) is now mandated and clients must implement at least one of the two.
+* The recommendation for TLS-X509-FINGERPRINT ({{RFC6614, Section 2.3}}) is removed since there are no RadSec implementations supporting it.
```

Comment:
```suggestion
* The recommendation for TLS-X509-FINGERPRINT ({{RFC6614, Section 2.3}}) is removed since the model has not been implemented by any known implementation of the experimental RADIUS/(D)TLS specifications.
```

We've just changed